src/diffusers/configuration_utils.py

```diff
@@ -134,10 +134,9 @@ def from_config(cls, pretrained_model_name_or_path: Union[str, os.PathLike], ret
                 The specific model version to use. It can be a branch name, a tag name, or a commit id, since we use a
                 git-based system for storing models and other artifacts on huggingface.co, so `revision` can be any
                 identifier allowed by git.
-            mirror (`str`, *optional*):
-                Mirror source to accelerate downloads in China. If you are from China and have an accessibility
-                problem, you can set this option to resolve it. Note that we do not guarantee the timeliness or safety.
-                Please refer to the mirror site for more information.
+            subfolder (`str`, *optional*, defaults to `""`):
+                In case the relevant files are located inside a subfolder of the model repo on huggingface.co, you can
```

Comment:
```suggestion
                In case the relevant files are located inside a subfolder of the model repo (either remote in huggingface.co or downloaded locally), you can
```

Reply:
Ah yeah good point!